content/themes/wcmc-beijing/blocks/carousel-full-width/block.php

```diff
@@ -7,4 +7,61 @@
  */
 
   // TODO: Build Gutenberg version
+
+  $index = 0;
 ?>
+
+<div class="carousel carousel--full-width">
+
+  <?php if ( block_rows( 'cells' ) ) : ?>
+
+    <div class="carousel__cells">
+
+      <?php while ( block_rows( 'cells' ) && ( $index < 1 )) : block_row( 'cells' );?>
+
```

Comment:
Is this only meant to be run once? I see `$index` initialized to 0 and `$index++` at the end of the loop, and part of this check is `$index < 1`